### PR TITLE
Refactor getNetwork code

### PR DESCRIPTION
**Pull request by arun1595**
##### Description

Refactor `switch` statements with an object in finding the network.

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

### CODE REVIEW

**Comment by owocki** [comment]
code LGTM so far... we might also want to replace the other `getNetwork()`s across the codebase

File: app/assets/ens/pending.js [hunk 1]
```diff
@@ -10,6 +10,19 @@ var waitforWeb3 = function(callback) {
...
 
+function getNetwork() {
+  var networks = {
+    '1': 'mainnet',
+    '2': 'morden',
+    '3': 'ropsten',
+    '4': 'rinkeby',
+    '42': 'kovan'
+  };
+  var customNetwork = 'custom network';
```
Comment (thelostone-mc) [inline app/assets/ens/pending.js]: worth directly returning this than storing it in a variable?

File: app/assets/ens/pending.js [hunk 2]
```diff
@@ -10,6 +10,19 @@ var waitforWeb3 = function(callback) {
   }
 };
 
+function getNetwork() {
```
Comment (thelostone-mc) [inline app/assets/ens/pending.js]: `getNetwork()` -> `getNetwork(id)`

File: app/assets/ens/pending.js [hunk 3]
```diff
@@ -18,29 +31,8 @@ window.addEventListener('load', function() {
     if (typeof web3 != 'undefined') {
       web3.version.getNetwork((error, netId) => {
         if (!error) {
+          var network = getNetwork();
```
Comment (thelostone-mc) [inline app/assets/ens/pending.js]: `getNetwork()` ->`getNetwork(netId)` ?

File: app/assets/ens/pending.js (hunk 2 above)
Comment (arun1595) [reply app/assets/ens/pending.js]: :see_no_evil: Fixed

**Comment by arun1595** [comment]
@thelostone-mc I missed @owocki 's comment. Let me do it :slightly_smiling_face:

**Comment by arun1595** [comment]
@thelostone-mc Done. I've replaced the fat arrow functions with traditional functions as well to address the rollbar issues we're getting related to using `=>`

Let me know if I missed something. Thanks!

**Pull request merged**